src/main/scala/firrtl/options/phases/WriteOutputAnnotations.scala

```diff
@@ -27,10 +26,21 @@ class WriteOutputAnnotations extends Phase {
   /** Write the input [[AnnotationSeq]] to a fie. */
   def transform(annotations: AnnotationSeq): AnnotationSeq = {
     val sopts = Viewer[StageOptions].view(annotations)
-    val serializable = annotations.filter{
-      case _: Unserializable    => false
-      case _: DeletedAnnotation => sopts.writeDeleted
-      case _                    => true
+    val serializable: AnnotationSeq = annotations.flatMap{
+      case _: Unserializable    => None
+      case a: DeletedAnnotation => if (sopts.writeDeleted) { Some(a) } else { None }
+      case a: HowToSerialize    =>
```

Comment:
I think especially in this context the name `SerializeToFile` would make more sense since other annotations also get serialized, but not to a custom file.